lib/Jsrt/ChakraCommon.h

```diff
@@ -225,6 +225,10 @@ typedef unsigned short uint16_t;
         /// </summary>
         JsErrorModuleParsed,
         /// <summary>
+        ///     Module was not yet evaluated when JsGetModuleNamespace was called.
+        /// </summary>
+        JsErrorModuleNotEvaluated,
```

Comment:
Since this is just an `enum` and not an `enum class`, I believe the compiler simply aliases it to an integer type in both C and C++.  So it should remain both source- and binary-compatible as long as the new constant is added to the end.  Adding it to the middle as done here is indeed problematic as it causes everything after it to be renumbered.

Reply:
Correct.  Adding it at the end will solve the problem

Reply:
I've moved it to the end of the section it's in - as it's the section for incorrect usage of API e.g. wrong parameters - I believe the numbering is restarted for each section (as the section headings have explicit values so this should be ok?

Reply:
Seems like this location works. Our new code is using a value which was previously unused and no other values in the enum are affected. I think @curtisman was implying to add to the end of the enum in his comment but if he's onboard with this we should be good.

Reply:
I didn't notice the start of the next section with an explicit value.
Yep.  Where you have now at the end of the section is good.   Thanks.